Feed/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/dist/css/styles.css">
+    <script src="https://kit.fontawesome.com/eed746e6fc.js" crossorigin="anonymous"></script>
+    <title>ChatApp - Feed</title>
+</head>
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+            <a class="navbar-brand" href="#">Navbar</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+          
+            <div class="collapse navbar-collapse" id="navbarSupportedContent">
+              <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                  <a class="nav-link" href="/profile/index.html">Profile <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
+                  <a class="nav-link" href="/Feed/index.html">Feed</a>
+                </li>
+              </ul>
+              <form class="form-inline my-2 my-lg-0">
+                <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+                <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+              </form>
+            </div>
+          </nav>
+    </header>
+    <main>
+        <div class="container-12 d-flex justify-content-center mt-5">
+            <div class="row">
+                <div class="col">
+                    <!-- Create Post Section -->
+                    <div class="card mb-4 d-flex justify-content-center">
+                        <div class="card-header">
+                            <h5>What's on your mind today?</h5>
+                        </div>
+                        <div class="card-body">
+                            <textarea class="form-control" rows="3" placeholder="Write your post..."></textarea>
+                        </div>
+                        <div class="card-footer">
+                            <button class="btn btn-primary">Create Post</button>
+                        </div>
+                    </div>
+                    <div class="input-group mb-3 ">
+                        <select class="custom-select" id="inputGroupSelect01">
+                            <option selected>Sort by</option>
+                            <option value="1">Date</option>
+                            <option value="2">A-Z</option>
+                            <option value="3">Newest posts</option>
+                        </select>
+                    </div>
+                    <!-- Posts -->
+                    <div class="card mb-4">
```

Comment:
According to the requirements stated in the brief, you need to have more cards here.